features/steps/steps.py

```diff
@@ -211,14 +239,59 @@ def instance_getter(i,representation_id, representation_type, negative=False):
         if condition(i, representation_id, representation_type):
             return i
 
+def strip_split(stmt, strp = ' ', splt = ' '):
+    return list(
+        map(str.lower, map(lambda s: s.strip(strp), stmt.split(splt)))
```

Comment:
It's easier to write `stmt.lower().split(splt)` then to use another map() for it after splitting

Reply:
WIP

Reply:
https://github.com/buildingSMART/ifc-gherkin-rules/pull/35